templates/src/objective-c/BoringSSL-GRPC.podspec.template

```diff
@@ -1558,5 +1560,2976 @@
     # symbols are src/objective-c/grpc_shadow_boringssl_symbol_list.
     # This is the last part of this file.
     s.prefix_header_contents = 
-      ${expand_symbol_list(settings.grpc_shadow_boringssl_symbols)}
```

Comment:
The original idea is that the `expand_symbol_list` will generate the list of symbols rather than you filling them in yourself. Did the generated list not work?

Reply:
Let me check, it was a different result.

Reply:
@muxi Yea you were right. I check it and all the symbols were equals. So try to regenerate it.